praw/exceptions.py

```diff
@@ -81,6 +142,33 @@ def parse_exception_list(
             )
             for exception in exceptions
         ]
+        for exception in baselist:
```

Comment:
Instead of this flag to differentiate between different cases of `RedditErrorItem` could we instead use different classes with the same interface?

Reply:
I don’t see the need to use different classes, as the only difference is the `__repr__` method.